app/plugin_api.go

```diff
@@ -725,6 +725,12 @@ func (api *PluginAPI) CreateBot(bot *model.Bot) (*model.Bot, *model.AppError) {
 	if bot.OwnerId == "" {
 		bot.OwnerId = api.id
 	}
+	// Bots cannot be owners of other bots
+	if user, err := api.app.GetUser(bot.OwnerId); err == nil {
+		if user.IsBot {
```

Comment:
Generally, plugins can do what they want. We don't do any permission checking usual. I'm 2/5 on removing the check here.

Reply:
If it was 3/5 I'd change it without question.  Since 2/5...   Why not leave it in?  It keeps out the bad 'state' of a bot being owned by another bot... unless that state is acceptable?

Reply:
I think this is about keeping the state of bots consistent. I don't think we should get bots be owners of other bots in any case.